docs/source/blog.rst

```diff
@@ -0,0 +1,69 @@
+Blog
+====
+
+We publish on `blog.jupyter.org` using the Ghost blog platform. Here are some
+guideline and warnings for authors using our blog platform either regularly, or
+as guests. 
+
+If you have any doubt, ask someone that have already published, some things
+cannot be easily undone. 
+
+You want to publish something on our blog ? Feel free to ask us, tell us what
+you want to write about. Hope fully it's Jupyter related ! We are always happy
+to get new content. If you don't feel comfortable writing on your own, we can
+alway help you to write you post. 
+
+
+Insertion of images:
+--------------------
+
+Try not to link to external images, if you want to put an image in the post,
+insert ``![]()`` in the editor and drag and drop from your desktop into the new
+created field. External images can change, and can break the blog post if they
+are taken down, if you drag and drop images they will be served from the same
+Content delivery network as the blog, which will insure the best experience for
+our readers.
+
+The image you see at the  top of our blog posts are set from within the
+metadata field, not using the `![]()`. They are treated differently that
+inlined images by many feeds reader (especially on mobile) and allow user on
+slow connection  to read the content of the blog earlier, which is a much
+better experience.  
+
+
+Links
+-----
+
+Do not use minified links when possible, the multiple redirect of minified
+links make the Mobile browsing experience a pain. If you need analytics of
+number of page view, this is tracked by Google analytics. 
+
+
+Posts Title
+-----------
+
+Alway check in the metadata fields that a blog post has a title and a canonical
+URL, is possible put the date in the canonical URL, in particular for events
+like jupyter-day, that can occur several time. The date of the even can differ
+from the date of the blog post. 
+
+Once you publish a Post, **never** change the title or the url, as it will
+break links of tweets, and RSS feeds. Keep in mind that when publishing some
+platform cache the url immediately, so changing the title will leads a lot of
+people to hit a 404 page. 
+
+Posts Updates
+-------------
+
+People that are subscribe to the blog might receive notification at every
+update of the post that is made. So use updates and fixes parsimoniously.
+It is OK to wait a few hours to fix a typo. 
+
+Always ask for a review
+-----------------------
+
+Once you think you are done, ask someone else to reread your post, and check
+the various parameters that you might have forgotten before publishing. 
+You are not on your own, this is teamwork !
+If we do things in a hurry you will probably spend more time fixing mistakes 
+that actually doing things right in a first place. 
```

Comment:
Ok, I'll try to rephrase all that tomorrow. I gave you access to my repo in case you want to push a fix directly.